Makefile

```diff
@@ -5,12 +5,25 @@ OT1_VERSION := 2.5.2
 OT2_VERSION_TAG := v3.3.1-beta.0
 OT2_MONOREPO_DIR := ot2monorepoClone
 
-OT1_FILE_SUFFIX := *.ot1.py
-OT2_FILE_SUFFIX := *.ot2.py
+# Parsers output to here
+BUILD_DIR := protoBuilds
+
+# Ignore all protocol dirs that contain a file named '.ignore'
+# on the top protocol folder level
+IGNORED_INPUT_PATHS := $(addsuffix %, $(dir $(wildcard protocols/*/.ignore)))
+
+OT1_INPUT_FILES_UNFILTERED := $(shell find protocols/ -type f -name '*.ot1.py')
+OT1_INPUT_FILES := $(filter-out $(IGNORED_INPUT_PATHS), $(OT1_INPUT_FILES_UNFILTERED))
+OT1_OUTPUT_FILES := $(patsubst protocols/%.ot1.py, $(BUILD_DIR)/%.ot1.py.json, $(OT1_INPUT_FILES))
+
+OT2_INPUT_FILES_UNFILTERED := $(shell find protocols/ -type f -name '*.ot2.py')
+OT2_INPUT_FILES := $(filter-out $(IGNORED_INPUT_PATHS), $(OT2_INPUT_FILES_UNFILTERED))
+OT2_OUTPUT_FILES := $(patsubst protocols/%.ot2.py, $(BUILD_DIR)/%.ot2.py.json, $(OT2_INPUT_FILES))
 
 .PHONY: install
 install:
-	python -m pip install virtualenv
+	python -m pip install virtualenv && \
+	make venvs/ot1 venvs/ot2
```

Comment:
I _think_ convention might be to use `$(MAKE)` rather than `make`. By convention I mean that "all the stackoverflow answers I find use it" 🙃